js/scenes/Map.js

```diff
@@ -43,8 +59,17 @@ const styles = StyleSheet.create({
     fontSize: 22,
     textAlign: 'center',
   },
-
   text: {
     ...defaultTextStyle,
   },
+  container: {
+    ...StyleSheet.absoluteFillObject,
+    height: 300,
+    width: 400,
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+  },
+  map: {
+    ...StyleSheet.absoluteFillObject,
+  },
 });
```

Comment:
It would be nice to have our chosen map component bundled inside a `<MagnetMap>` component that exposes just the APIs we need and can be used across the app :)